pages.es/linux/groupdel.md

```diff
@@ -1,6 +1,7 @@
 # groupdel
 
 > Elimina del sistema grupos de usuarios existentes.
+> Más información: <https://manned.org/groupdel.8>.
```

Comment:
```suggestion
> Más información: <https://manned.org/groupdel>.
```
`manned.org` doesn't need sections in the link. (That also applies to the other links).

Reply:
ah ok, my mystake, I'll do another batch right away then. Cheer's